src/synthesizer/utils/geometry.py

```diff
@@ -0,0 +1,69 @@
+"""A module containing geometry functions.
+
+Example usage:
+
+    import numpy as np
+    from geometry import get_rotation_matrix
+
+    # Define the vector to rotate
+    vector = np.array([1.0, 0.0, 0.0])
+
+    # Define the unit vector to rotate to
+    unit = np.array([0.0, 1.0, 0.0])
+
+    # Get the rotation matrix
+    matrix = get_rotation_matrix(vector, unit)
+"""
+
+import numpy as np
+
+
+def get_rotation_matrix(vector, unit=None):
```

Comment:
The method is taken directly from stackoverflow, would be nice to quote that: https://stackoverflow.com/questions/43507491/imprecision-with-rotation-matrix-to-align-a-vector-to-an-axis

Reply:
Included